libs/content/workspace/src/tab/mod.rs

```diff
@@ -535,7 +535,10 @@ pub fn canonicalize_path(path: &str) -> String {
     result.to_string_lossy().to_string()
 }
 
-pub fn core_get_by_relative_path(core: &Lb, from: Uuid, path: &Path) -> Result<File, String> {
+pub fn core_get_by_relative_path<P: AsRef<Path>>(
+    core: &Lb, from: Uuid, path: P,
+) -> Result<File, String> {
+    let path = path.as_ref();
```

Comment:
is this validated? this is how we decide if the url is relative or not, ultimately

Reply:
lb-rs will be the thing that does the final check on the resolved url, if the thing they click, after processing `../` and `/` is a valid path in lockbook, we'll open that UUID, otherwise we don't do anything and let the OS take it from there.